webpack/webpack.config.cjs

```diff
@@ -286,6 +286,7 @@ module.exports = function(env, argv) {
     ],
     stats: {
       children: true,
+      errorDetails: true,
```

Comment:
Does this likewise need to be in the [dev-server configuration](https://webpack.js.org/configuration/dev-server/#devserverstats-)? Or is this intended to help in debugging the gh-pages build?

Reply:
This entire `stats` key would go in the dev server config, but that change isn't in main yet.

Reply:
I guess another way to ask this: Why is this being added? I'm definitely not opposed, just curious what it gets us/what problem you were solving

Reply:
When child components (in this case the worker for raidemulator) get built, the `children` flag allows general error information, and the `errorDetails` flag allows a full stack if there's a compile error when building.

In this case it helped me track down a specific issue with the manifest change that wasn't visible without the flag enabled.

Reply:
Sorry, I see now what you're asking. This is something else, not what you linked, I believe.

https://webpack.js.org/configuration/stats/

Reply:
That sounds reasonable. Thanks!